verible/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+set -x
+
+sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
+sudo apt update
+sudo apt install -y gcc-8 g++-8
+
+export CC=gcc-8
+export CXX=g++-8
+
+wget https://github.com/bazelbuild/bazel/releases/download/0.29.1/bazel-0.29.1-installer-linux-x86_64.sh
+chmod +x bazel-0.29.1-installer-linux-x86_64.sh
+./bazel-0.29.1-installer-linux-x86_64.sh --prefix=$PREFIX
```

Comment:
I was unable to build it locally with bazel 0.20.0 (which is available as a conda package - this installing this by hand like that).